Documentation/ProjectFile/prj/process_variables/process_variable/t_initial_condition.md

```diff
@@ -1 +1,3 @@
-\ogs_missing_documentation
+Initial condition for the process variable.
+
+This setting references a \ref ogs_file_param__parameter "parameter" by name.
```

Comment:
I added a few words, but I didn't mention the linear solver. I think that belongs better to the time loop description.